analyzer/src/main/kotlin/managers/GoDep.kt

```diff
@@ -272,7 +273,11 @@ class GoDep(
 
         val repoRoot = Paths.get(gopath.path, "src", importPath).toFile()
 
+        // Create fallback VcsInfo for known VCS hosts.
+        val fallbackVcsInfo =
+            VcsHost.toVcsHost(URI("https://$importPath"))?.toVcsInfo("https://$importPath") ?: VcsInfo.EMPTY
```

Comment:
I explicitly did not use `VcsHost.toVcsInfo(importPath)`, as this would also try to create `VcsInfo`s for non-VCS URLs.
E.g. 
```yml
        vcs_processed:
          type: ""
          url: "go.uber.org/atomic"
          revision: "1ea20fb1cbb1cc08cbd0d913a96dead89aa18289"
          path: ""
```

Reply:
Hmm, that might even be a bug in `VcsHost.toVcsInfo()`. I guess I'd use it anyway as it's shorter, but only take the return value if it's not a `VcsType.UNKNOWN`.